servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/DefaultNettyConnection.java

```diff
@@ -237,6 +231,7 @@ private DefaultNettyConnection(Channel channel, BufferAllocator allocator, Execu
      * @param idleTimeoutMs Value for {@link ServiceTalkSocketOptions#IDLE_TIMEOUT IDLE_TIMEOUT} socket option.
      * @param initializer Synchronously initializes the pipeline upon subscribe.
      * @param executionStrategy {@link ExecutionStrategy} to use for this connection.
+     * @param protocol The {@link Protocol} this {@link DefaultNettyConnection} implements.
```

Comment:
As this connection does not implement any protocol per se, I would suggest:

```suggestion
     * @param protocol {@link Protocol} for the returned {@link DefaultNettyConnection}.
```